implementations/rust/ockam/ockam_command/src/service/list.rs

```diff
@@ -0,0 +1,40 @@
+use clap::Args;
+use ockam::{Context, TcpTransport};
+
+use ockam_api::nodes::models::services::ServiceList;
+
+use crate::util::{api, node_rpc, RpcBuilder};
+use crate::CommandGlobalOpts;
+
+/// List service(s) of a given node
+#[derive(Clone, Debug, Args)]
+pub struct ListCommand {
+    /// Name of the node.
+    #[arg(display_order = 900)]
+    pub node_name: String,
```

Comment:
This argument should be like:

```rust
#[command(flatten)]
pub node_opts: NodeOpts,
```

so the commands looks like: `service list --node {name}`, or `service list` defaulting to the default node name.

Reply:
Solved in latest commit - I went with `NodeOpts` before but took it out right before making the initial PR :sweat_smile:.